test/framework/easyconfig.py

```diff
@@ -1977,6 +1981,7 @@ def test_dump_comments(self):
             '# this is a header',
             '#',
             '# which may include empty comment lines',
+            '# weirdly indented lines',
```

Comment:
A bit of nitpicking, this should really keep the initial spaces, or?

Reply:
No, I disagree... The indent is wrong initially, and it's OK to have it properly indented in the dumped easyconfig.

Reply:
Ok